cmd/skeleton-recommender/main.go

```diff
@@ -70,17 +74,23 @@ func autoscaling(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	log.Printf("< %+v\n", request)
 	log.Printf("> %+v\n\n", response)
 }
 
+// Redirects /autoscaling/redirect to /autoscaling
```

Comment:
What's the purpose of this redirect?

Reply:
It's to test that redirects work, as they will be needed by most implementations (most of the time you end up having to redirect to the specific shard of the control plane that can give you the recommendation)